build-scripts/split-tests.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+set -e
+
+# ----------------- Functions ------------------
+
+function list_tests() {
+    local build_tags=$1
+    local pkg=$2
+
+    go test -tags="${build_tags}" -list . "${pkg}" | grep "^Test" | sort
+}
+
+function split_list() {
+    local chunks_count=$1
+    local input_file=$2
+    local output_dir=$3
+
+    local total_lines
+    local lines_per_file
+
+    total_lines=$(wc -l < "${input_file}" | xargs)
+    ((lines_per_file = (total_lines + chunks_count - 1) / chunks_count))
+
+    echo ">> Tests total count: ${total_lines}"
+    mkdir -p "${chunks_dir}"
```

Comment:
Is `chunks_dir` supposed to be a reference to the local `output_dir` variable?

Reply:
Addressed in https://github.com/hashicorp/consul-terraform-sync/pull/869/commits/cc62f5479ab0303000a51d0e40c7a9a8fb3790bb